hasura/metadata/databases/default/tables/public_vault_transactions.yaml

```diff
@@ -2,48 +2,33 @@ table:
   name: vault_transactions
   schema: public
 object_relationships:
-- name: user
+- name: circle
+  using:
+    foreign_key_constraint_on: circle_id
+- name: distribution
+  using:
+    foreign_key_constraint_on: distribution_id
+- name: profile
   using:
     foreign_key_constraint_on: created_by
 - name: vault
   using:
     foreign_key_constraint_on: vault_id
-insert_permissions:
-- permission:
-    backend_only: false
-    check:
-      vault:
-        protocol:
-          circles:
-            users:
-              profile:
-                id:
-                  _eq: X-Hasura-User-Id
-    columns:
-    - id
-    - name
-    - tx_hash
-    - description
-    - value
-    - date
-    - vault_id
-    - created_by
-    - created_at
-    - updated_at
-  role: user
+- name: vault_tx_type
+  using:
+    foreign_key_constraint_on: tx_type
 select_permissions:
 - permission:
     columns:
+    - circle_id
+    - created_at
+    - created_by
+    - distribution_id
     - id
-    - name
     - tx_hash
-    - description
-    - value
-    - date
-    - vault_id
-    - created_by
-    - created_at
+    - tx_type
     - updated_at
+    - vault_id
     filter:
```

Comment:
`vaults` select permissions check whether `users` `deleted_at` `is_null` might want to be consistent here also